mlflow/tracking/fluent.py

```diff
@@ -127,6 +127,37 @@ def start_run(run_id=None, experiment_id=None, run_name=None, nested=False):
     :param nested: Controls whether run is nested in parent run. ``True`` creates a nest run.
     :return: :py:class:`mlflow.ActiveRun` object that acts as a context manager wrapping
              the run's state.
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        # Create nested runs
+        with mlflow.start_run(run_name='PARENT_RUN') as parent_run:
+            mlflow.log_param("parent", "yes")
+            with mlflow.start_run(run_name='CHILD_RUN', nested=True) as child_run:
+                mlflow.log_param("child", "yes")
+
+        print("parent run_id: {}".format(parent_run.info.run_id))
+        print("child run_id : {}".format(child_run.info.run_id))
+        print("--")
+
+        # Search all child runs with a parent id
+        query = "tags.mlflow.parentRunId = '{}'".format(parent_run.info.run_id)
+        results = mlflow.search_runs(filter_string=query)
+
+        # Print the pandas DataFrame columns
+        print(results[["run_id", "params.child", "tags.mlflow.runName"]])
```

Comment:
```suggestion
        results = mlflow.search_runs(filter_string=query)
        print(results[["run_id", "params.child", "tags.mlflow.runName"]])
```

because it's obvious we print `results`.